### ISSUE TITLE
js-properties gives false positive on init

### ISSUE THREAD

**Issue by tomByrer** [mathiasbynens/mothereff.in#54]
When I first visit in Firefox, it gives incorrect green results for `foo-bar`:

![image](https://user-images.githubusercontent.com/1308419/89035512-b0ecef80-d300-11ea-87fd-14ba7731919e.png)

---

But when I edit it to `foo-ba`, it gives the correct results:

![image](https://user-images.githubusercontent.com/1308419/89035608-ded23400-d300-11ea-8561-1fadf5654f66.png)

Other than that small bug, it is very helpful, cheers!

**Comment by janbrasna** [comment]
Yup, present in current chromium too — the sample default after loading the tool is not the same state (green+yellow) as after editing the placeholder back and forth to the same identifier name (orange+red).
